<launch>
    <arg name="vehicle" default="icab1" />

    <node pkg="hypergrid" type="laser_to_gridmap" name="laser_to_gridmap" output="screen">
        <rosparam command="load" file="$(find hypergrid)/config/laser_to_gridmap.yaml"/>
        <param name="map_frame_id" value="$(arg vehicle)/base_footprint" />
        <remap from="scan" to="/$(arg vehicle)/scan" />
    </node>
</launch>
